docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,188 @@
+ZAAKTYPEN API
+===============================================
+**FUNCTIONELE DOCUMENTATIE**
+
+[aanzet tot documentatie van de API’s van het ZDS2-koppelvlak. Is vooralsnog een
+‘try-out’ n.a.v. [userstory
+\#188](https://github.com/VNG-Realisatie/gemma-zaken/issues/188) om inzicht te
+krijgen in de wijze van (functioneel) documenteren van API’s, als onderdeel van
+koppelvlakdocumentatie]
+
+| **Aspect**      | **Beschrijving**                                                                                                                                                                                  |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Doel            | Het kunnen aanspreken van een voorziening voor het onderhouden en raadplegen van zaaktypecatalogus (ZTC) inclusief daarinopgenomen zaaktypen, statustypen, roltypen en informatieobjecttypen, rollen en relaties naar betrokkenen, objecten, zaaktypen en statustypen. |
```

Comment:
Ik heb het accent op de zaaktypen gelegd, omdat het m.i. daar om draait. En de details weggelaten. En van het laatste deel ('relaties naar ..') is het maar de vraag of dat terecht is, lijkt me van niet.